cas-server-core-events-jpa/src/main/java/org/jasig/cas/support/events/jpa/JpaCasEventRepository.java

```diff
@@ -0,0 +1,48 @@
+package org.jasig.cas.support.events.jpa;
+
+import org.jasig.cas.support.events.dao.AbstractCasEventRepository;
+import org.jasig.cas.support.events.dao.CasEventDTO;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.annotation.PostConstruct;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.validation.constraints.NotNull;
+import java.util.Collection;
+
+/**
+ * This is {@link JpaCasEventRepository} that stores event data into a RDBMS database.
+ *
+ * @author Misagh Moayyed
+ * @since 4.3.0
+ */
+@Repository("casEventRepository")
+@Transactional
+public class JpaCasEventRepository extends AbstractCasEventRepository {
+
+    @NotNull
+    @PersistenceContext(unitName = "eventsEntityManagerFactory")
+    private EntityManager entityManager;
+
+    /**
+     * Initialized post construction.
+     */
+    @PostConstruct
+    public void init() {}
```

Comment:
Must have been a bad copy, but it may become useful as an extension point later. I'll remove it for now.